src/services/GradingService.js

```diff
@@ -0,0 +1,96 @@
+
+import request from '../request'
+
+function getImportMethod(sheetLink) {
+
+    var parse = require('url-parse')
+      , url = parse(sheetLink, true);
+    if (url.hostname === "alexuuni-my.sharepoint.com") {
+      return "import-ms"
+  
+    } else if (url.hostname === "docs.google.com") {
+      return "import-google"
+  
+    }
+  
+  
+  }
+
+function getLabs(course) {
+    return request({
```

Comment:
As far as I remember, axios has an `axios.get` / `axios.post` methods. I'm just hinting so feel free to keep the code here as is with the `method` parameter